### ISSUE TITLE
DPL-105 As a provided of user support I would like a USG actions page [OKR] [C=M,V=4]

### ISSUE THREAD

**Issue by JamesGlover** [sanger/sequencescape#3279]
**User story**
As a member of USG and PSD I would like a USG actions tab to perform common support actions from a web interface with proper tracking. We should begin with the most common actions that aren't better serviced by self-service approaches elsewhere.

**Who are the primary contacts for this story**
Neil S
James G

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] Provide a USG actions tab similar to UAT actions
- [ ] Use the UAT actions system as a basis for adding new actions
- [ ] Actions should be easily testable
- [ ] Add a PerfomedActions record that tracks:
  - RT number associated with the action (Consider also allowing GH issues)
  - User who performed the action
  - Version + SHA of Sequencescape at time action was performed
  - Additional comment field
- [ ] Performed action should have polymorphic has-many association with affected records

***Nice to haves***
- Ability to configure new issue trackers, and map the issue id back to a URL
- Show page for the PerformedActions
- Easily link modified records to performed actions. A shared partial/component we can drop in anywhere would be good here. Let us add something like `<%= usg_actions_for(records) %>` to a view and be done with it.

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by JamesGlover** [comment]
https://github.com/sanger/sequencescape/issues/3392
